calblueprint/meepanyar
# [Translation Support] Apply translations (screens/Camera)

# In this PR

In this PR, we follow the steps in the ["Applying Our Hook" section](https://www.notion.so/calblueprint/Translation-Support-Design-Document-7703018a8c4e403dab8aaa49d2a322e7#a7e5cbd6962e4dbd850b724c3eee7b18) of our [roadmap for translation support](https://www.notion.so/calblueprint/Translation-Support-Design-Document-7703018a8c4e403dab8aaa49d2a322e7#cfc52fc0865e4bb2b140e54d395343ba) in order to translate `screens/Camera` for `meepanyar`. 

## Coverage
- [ ] `screens/Customers`

## How to Review
1. Change your browser device setting to Burmese (or return "my" in `device.ts#detectLanguage`)
2. Run the application and navigate to the screens corresponding to `screens/Camera`. 
3. Ensure nothing is breaking
4. Test in English and ensure wording is still correct

We will translate the rest of `screens/*` in subsequent PRs. 

cc: @julianrkung
